Automattic/jetpack
# E2E tests: Try e2e allure reporting

This PR adds the Allure2 dashboard, and related results generation into the Travis CI pipeline. 
With this merged, we'll be able to present E2E results in a more clean, accessible way via static HTML page: https://brbrr.github.io/test-dashboard-pages/ (We might move the dashboard somewhere else later on)

site's root (aka `/`) will serve test results for `master` branch
while PR and branch results would be served as sub-folders: `/pr-XXXX` & `add_fancy-new-feature` (NOTE: all `/` are replaced with `_`) 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* If you're an Automattician, include a shortlink to the p2 discussion with Jetpack Product here.
https://wp.me/p9dueE-1lv

#### Testing instructions:






* visit https://brbrr.github.io/test-dashboard-pages/pr-15391, and click around
* Try restarting the build, it should generate a new report, which you can check by going to Suites > Select a suite > Retries (in the right column)

#### Proposed changelog entry for your changes:

* n/a
